docs/docs/coverage/language/python.md

```diff
@@ -24,10 +24,10 @@ The following table provides an outline of the features Trivy offers.
 
 | Package manager | File             | Transitive dependencies | Dev dependencies | [Dependency graph][dependency-graph] | Position | [Detection Priority][detection-priority] |
 |-----------------|------------------|:-----------------------:|:----------------:|:------------------------------------:|:--------:|:----------------------------------------:|
-| pip             | requirements.txt |            -            |     Include      |                  -                   |    ✓     |                    ✓                     |
-| Pipenv          | Pipfile.lock     |            ✓            |     Include      |                  -                   |    ✓     |                Not needed                |
-| Poetry          | poetry.lock      |            ✓            |     Exclude      |                  ✓                   |    -     |                Not needed                |
-| uv              | uv.lock          |            ✓            |     Exclude      |                  ✓                   |    -     |                Not needed                |
+| pip             | requirements.txt |            -            |     [Include](#pip)      |                  -                   |    ✓     |                    ✓                     |
+| Pipenv          | Pipfile.lock     |            ✓            |     [Include](#pipenv)      |                  -                   |    ✓     |                Not needed                |
+| Poetry          | poetry.lock      |            ✓            |     [Exclude](#poetry)      |                  ✓                   |    -     |                Not needed                |
+| uv              | uv.lock          |            ✓            |     [Include](#uv)      |                  ✓                   |    -     |                Not needed                |
```

Comment:
you only need to add links if the `--include-dev-deps` flag is used for the file

Reply:
In nodejs all package managers have links. I think the documentation should be consistent.

Reply:
Then we should update the nodejs documentation

Reply:
Fixed

Reply:
> In nodejs all package managers have links. I think the documentation should be consistent.

I missed that a link was added for `Bun`. We can remove the link for that.
But for other files - `yarn`, `npm` and `pnpm` support `--include-dev-deps`. That's why they have links.

Reply:
OK, I'll open a PR for that.